### ISSUE TITLE
Type definitions export ES Modules but should export CommonJS

### ISSUE THREAD

**Issue by OliverJAsh** [archfz/cypress-terminal-report#151]
Reduced test case: https://github.com/OliverJAsh/cypress-10-cypress-terminal-report-errors

This is a brand new Cypress project.

Run `yarn` and then `tsc`.

```sh-session
❯ tsc
cypress.config.js:7:13 - error TS2349: This expression is not callable.
  Type 'typeof import("/Users/oliverash/Development/temp/cypress-10-cypress-terminal-report-errors/node_modules/cypress-terminal-report/src/installLogsPrinter")' has no call signatures.

7             enableTerminalReportLogsPrinter(on);
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in cypress.config.js:7
```

This is happening because the type definitions are defined using ES Modules (`export default installLogsPrinter;`) however they should be defined as CommonJS (`export = installLogsPrinter;`) to match the JS source code.

https://github.com/archfz/cypress-terminal-report/blob/c7d74ceefa94b76529b81e09354a746f1085eb40/src/installLogsPrinter.d.ts#L113

https://github.com/archfz/cypress-terminal-report/blob/c7d74ceefa94b76529b81e09354a746f1085eb40/src/installLogsPrinter.js#L324

https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

Related change: https://github.com/archfz/cypress-terminal-report/commit/59f0ab999b1e4b913ffa0ae81357f29960218ed1#diff-2a909c3c7c488c66f7e2c41d5b33f5c0f062d164a5a5e47485318e0697a0d7adL87

Related issue: https://github.com/archfz/cypress-terminal-report/issues/149

**Comment by OliverJAsh** [comment]
`esModuleInterop` is already enabled in the reduced test case  https://github.com/OliverJAsh/cypress-10-cypress-terminal-report-errors/blob/67f6a73665790988aedd108494be1ee8b6b63022/tsconfig.json#L6

**Comment by archfz** [comment]
Can you please check the demo project from readme, using this plugin? That is also with typescript and it's working. Check for difference against your setup.

**Comment by OliverJAsh** [comment]
The difference is the Cypress config file uses ES imports: https://github.com/archfz/cypress-terminal-report-demo/blob/7d3c107bf102d0280b01431b3f518c5169561f73/cypress.config.ts#L2

Whereas in my config file I'm using a CommonJS `require`: https://github.com/OliverJAsh/cypress-10-cypress-terminal-report-errors/blob/67f6a73665790988aedd108494be1ee8b6b63022/cypress.config.js#L2

I think this tool should support config files with CommonJS as this is supported by Cypress. I have explained a way to fix this above.

**Comment by archfz** [comment]
Ok thank you. PRs are welcome btw.

**Comment by archfz** [comment]
Released fix in 4.0.2. Please test.

**Comment by drebrez** [comment]
Hi @archfz ,

I've just noticed that this problem also applies to the installLogsCollector.

In commit https://github.com/archfz/cypress-terminal-report/commit/743b551d3a29d34ac8d256405525152b73b2a2c4 you fixed it only for the installLogsPrinter.

I've created a PR with the fix https://github.com/archfz/cypress-terminal-report/pull/167
